src/Assets/scss/custom/userProfile.scss

```diff
@@ -0,0 +1,56 @@
+
+@media screen and (max-width: 992px) {
+    .row {
+      display: block;
+    }
```

Comment:
be careful when modifying bootstrap classes. they may break other features

Reply:
fixed